api/v1/openapi.yaml

```diff
@@ -1834,6 +1837,24 @@ definitions:
         type: array
         items:
           type: string
+  Srv6:
+    description: |-
+      Status of SRv6 support
+
+      +k8s:deepcopy-gen=true
+    type: object
+    properties:
+      enabled:
+        type: boolean
+      devices:
+        type: array
+        items:
+          type: string
+      SRv6EncapMode:
```

Comment:
I believe this should be `srv6EncapMode:` for consistency with other properties.